lib/internal/modules/esm/get_format.js

```diff
@@ -61,6 +64,8 @@ function defaultGetFormat(url, context, defaultGetFormatUnused) {
           'The Node.js specifier resolution in ESM is experimental.',
           'ExperimentalWarning');
         format = legacyExtensionFormatMap[ext];
+      } else if (parsed.pathname.endsWith('/')) {
```

Comment:
```suggestion
      } else if (StringPrototypeEndsWith(parsed.pathname, '/')) {
```
and a corresponding destructure on line 2